CefSharp.OutOfProcess.Core/OutOfProcessHost.cs

```diff
@@ -85,11 +86,12 @@ public string ChromiumVersion
         /// <param name="handle">handle used to host the control</param>
         /// <param name="url"></param>
         /// <param name="id"></param>
+        /// <param name="requestContextPreferences">request context preference.</param>
```

Comment:
The comment needs to be updated to describe the actual behaviour, a new `RequestContext` sharing the same settings as the `Global`.